Client/src/Components/TerryComponent/index.jsx

```diff
@@ -1,7 +1,95 @@
-import React from "react";
+// eslint-disable-next-line no-unused-vars
+import React, { useState } from "react";
+import { Button, Container } from "@mui/material";
+import AlertMessage from "./reusable/AlertMessage";
 
-const index = () => {
-  return <div></div>;
+const Index = () => {
+  const [notification, setNotification] = useState({
+    open: false,
+    title: "",
+    message: "",
+    type: "",
+    alertMessageType: "",
+    actionText: "",
+  });
+
+  const handleOpenAlertMessage = (type, title, message, actionText) => {
+    setNotification({
+      ...notification,
+      open: true,
+      title,
+      type,
+      message,
+      actionText,
+    });
+  };
+
+  const handleCloseAlertMessage = () => {
+    setNotification({ ...notification, open: false });
+  };
+
+  const handleDismissText = () => {
+    setNotification({ ...notification, open: false });
+  };
+
+  return (
+    <div>
+      <Container style={{ marginTop: "100px", marginLeft: "70px" }}>
+        <Button
+          variant="contained"
+          onClick={() =>
+            handleOpenAlertMessage(
+              "release",
+              "We've just released a new feature",
+              "Lorem ipsum dolor sit amet consectetur adipiscing elit. Aliquid pariatur, ipsum dolor.",
+              "View changes"
+            )
+          }
+        >
+          Feature Alert
+        </Button>
+        <Button
+          style={{ margin: "10px 20px 10px 20px" }}
+          className="button"
+          variant="contained"
+          onClick={() =>
+            handleOpenAlertMessage(
+              "problem",
+              "There was a problem with that action",
+              "Lorem ipsum dolor sit amet consectetur adipiscing elit. Aliquid pariatur, ipsum dolor.",
+              "Learn more"
+            )
+          }
+        >
+          Problem Alert
+        </Button>
+        <Button
+          variant="contained"
+          onClick={() =>
+            handleOpenAlertMessage(
+              "update",
+              "Successfully updated profile",
+              "Lorem ipsum dolor sit amet consectetur adipiscing elit. Aliquid pariatur, ipsum dolor.",
+              "View changes"
+            )
+          }
+        >
+          Update Alert
+        </Button>
```

Comment:
There's a lot of duplicated code here, you could consider declaring the buttons in an array and then mapping them to a component.  Again not a hard and fast requirement, but it's always nice to reduce code repetition wherever possible. 

It makes the code cleaner to read and if we need to update the buttons we can do it all at once rather than 3 times seperately.  Something to keep in mind :+1: